File: Java/1109.java
import java.util.Scanner;

public class Main{	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		String name = sc.nextLine();
		int age = sc.nextInt();
		char dep = sc.next().charAt(0);
		float key = sc.nextFloat();
		
		System.out.println(name);
		System.out.println(age);
		System.out.println(dep);
		System.out.println(key);
	}
}

File: C/1109.c
#include <stdio.h>

int main() {
	char name[20];
	int age;
	char dep;
	float key;

	scanf("%s\n", name);
	scanf("%d\n", &age);
	scanf("%c\n", &dep);
	scanf("%g", &key);

	printf("%s\n", name);
	printf("%d\n", age);
	printf("%c\n", dep);
	printf("%g", key);

	return 0;
}